migrations/20170206131527_users.js

```diff
@@ -1,7 +1,7 @@
 exports.up = (knex) => {
   return knex.schema.createTable('users', (table) => {
     table.increments();
-    table.integer('linkedin_id').notNullable().unique();
+    table.string('linkedin_id').notNullable().unique();
```

Comment:
Aaahh, should have caught this. This is a no no dude. Gotta create a new migration.